WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -461,14 +463,48 @@ private extension OrderDetailsDataSource {
         cell.configure(item: itemViewModel, imageService: imageService)
     }
 
-    private func configureShippingLabelTrackingNumber(cell: OrderTrackingTableViewCell, at indexPath: IndexPath) {
+    private func configureShippingLabelTrackingNumber(cell: ImageAndTitleAndTextTableViewCell, at indexPath: IndexPath) {
         guard let shippingLabel = shippingLabel(at: indexPath) else {
             return
         }
 
-        cell.topText = shippingLabel.serviceName
+        let viewModel = ImageAndTitleAndTextTableViewCell.ViewModel(title: Title.shippingLabelTrackingNumberTitle,
+                                                                    text: shippingLabel.trackingNumber,
+                                                                    textTintColor: nil,
+                                                                    image: .locationImage,
+                                                                    imageTintColor: .primary,
+                                                                    numberOfLinesForTitle: 0,
+                                                                    numberOfLinesForText: 0,
+                                                                    isActionable: false)
+        cell.updateUI(viewModel: viewModel)
+
+        let actionButton = UIButton(type: .detailDisclosure)
+        actionButton.applyIconButtonStyle(icon: .moreImage)
+        actionButton.on(.touchUpInside) { [weak self] sender in
+            self?.onCellAction?(.shippingLabelTrackingMenu(shippingLabel: shippingLabel, sourceView: sender), nil)
+        }
+        cell.accessoryView = actionButton
+    }
+
+    private func configureShippingLabelRefunded(cell: ImageAndTitleAndTextTableViewCell, at indexPath: IndexPath) {
+        guard let shippingLabel = shippingLabel(at: indexPath), let refund = shippingLabel.refund else {
+            return
+        }
 
-        // TODO-2167: update design for shipping label tracking number
+        let title = String.localizedStringWithFormat(Title.shippingLabelRefundedTitleFormat, shippingLabel.serviceName)
+        let currencyFormatter = CurrencyFormatter(currencySettings: ServiceLocator.currencySettings)
```

Comment:
Do you think we should instantiate this as a `lazy var` instead of on each cell configuration? Creating formatters is normally expensive, especially on older devices!

Reply:
Great point! moved this to a lazy var in https://github.com/woocommerce/woocommerce-ios/pull/3226/commits/7d562f488ccadb35253d19990889883e13da5f25